data/yam/agama/auto/lib/storage.libsonnet

```diff
@@ -37,12 +37,42 @@ local lvm(encrypted=false) = {
         { generate: 'default' },
       ],
     },
-  ]
+  ],
+};
+
+local separate_home_boot() = {
+  drives: [
+    {
+      search: '/dev/vda',
+      filesystem: {
+        path: '/home',
+      },
+    },
+    {
+      search: '/dev/vdb',
+      partitions: [
+        {
+          filesystem: {
+            path: '/',
+          },
+        },
+      ],
+    },
+    {
+      search: '/dev/vdc',
+      alias: 'boot-disk',
+    },
+  ],
+  boot: {
+    configure: 'true',
+    device: 'boot-disk',
+  },
 };
 
 {
   lvm: lvm(false),
   lvm_encrypted: lvm(true),
   root_filesystem_ext4: root_filesystem('ext4'),
   root_filesystem_xfs: root_filesystem('xfs'),
+  separate_home_boot: separate_home_boot(),
```

Comment:
it is more than that, please call it `whole_disk_and_boot_partition`

Reply:
Changed.